2022-01-10 17:59:11,317 INFO    MainThread:17268 [wandb_setup.py:_flush():71] setting env: {}
2022-01-10 17:59:11,317 INFO    MainThread:17268 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-10 17:59:11,317 INFO    MainThread:17268 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20220110_175911-3e2zlrnx\logs\debug.log
2022-01-10 17:59:11,318 INFO    MainThread:17268 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20220110_175911-3e2zlrnx\logs\debug-internal.log
2022-01-10 17:59:11,318 INFO    MainThread:17268 [wandb_init.py:init():404] calling init triggers
2022-01-10 17:59:11,318 INFO    MainThread:17268 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-10 17:59:11,320 INFO    MainThread:17268 [wandb_init.py:init():449] starting backend
2022-01-10 17:59:11,320 INFO    MainThread:17268 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-10 17:59:11,325 INFO    MainThread:17268 [backend.py:ensure_launched():199] starting backend process...
2022-01-10 17:59:11,480 INFO    MainThread:17268 [backend.py:ensure_launched():204] started backend process with pid: 3360
2022-01-10 17:59:11,483 INFO    MainThread:17268 [wandb_init.py:init():458] backend started and connected
2022-01-10 17:59:11,484 INFO    MainThread:17268 [wandb_init.py:init():519] updated telemetry
2022-01-10 17:59:11,520 INFO    MainThread:17268 [wandb_init.py:init():549] communicating current version
2022-01-10 17:59:13,147 INFO    MainThread:17268 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-10 17:59:13,149 INFO    MainThread:17268 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-10 17:59:13,391 INFO    MainThread:17268 [wandb_init.py:init():592] starting run threads in backend
2022-01-10 17:59:13,644 INFO    MainThread:17268 [wandb_run.py:_console_start():1816] atexit reg
2022-01-10 17:59:13,646 INFO    MainThread:17268 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-10 17:59:13,646 INFO    MainThread:17268 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-10 17:59:13,651 INFO    MainThread:17268 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-10 17:59:13,651 INFO    MainThread:17268 [wandb_init.py:init():619] run started, returning control to user process
2022-01-10 17:59:13,651 INFO    MainThread:17268 [wandb_config.py:__setitem__():144] config set learning_rate = 0.001 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001DF01D16FD0>>
2022-01-10 17:59:13,651 INFO    MainThread:17268 [wandb_run.py:_config_callback():962] config_cb learning_rate 0.001 None
2022-01-10 17:59:13,651 INFO    MainThread:17268 [wandb_config.py:__setitem__():144] config set batch_size = 32 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001DF01D16FD0>>
2022-01-10 17:59:13,652 INFO    MainThread:17268 [wandb_run.py:_config_callback():962] config_cb batch_size 32 None
2022-01-10 17:59:14,470 INFO    MainThread:17268 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-01-10 17:59:14,470 INFO    MainThread:17268 [wandb_run.py:_restore():1758] restore
2022-01-10 17:59:14,695 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 980
}

2022-01-10 17:59:15,061 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 980
}

2022-01-10 17:59:15,428 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 6174
}

2022-01-10 17:59:15,532 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 6174
}

2022-01-10 17:59:15,643 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 980
  total_bytes: 6174
}

2022-01-10 17:59:15,748 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3806
  total_bytes: 6174
}

2022-01-10 17:59:15,852 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}

2022-01-10 17:59:15,962 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}

2022-01-10 17:59:16,068 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}

2022-01-10 17:59:16,181 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}

2022-01-10 17:59:16,288 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}

2022-01-10 17:59:16,467 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}

2022-01-10 17:59:17,027 INFO    MainThread:17268 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6174
  total_bytes: 6174
}
local_info {
}

2022-01-10 17:59:18,175 INFO    MainThread:17268 [wandb_run.py:_append_files():2184] logging synced files
